examples/aws/complete/main.tf

```diff
@@ -0,0 +1,48 @@
+
+locals {
+  cluster_endpoint                   = var.create ? module.agent_cluster.cluster_endpoint : data.aws_eks_cluster.my_eks[0].endpoint
+  cluster_certificate_authority_data = var.create ? module.agent_cluster.cluster_certificate_authority_data : data.aws_eks_cluster.my_eks[0].certificate_authority[0].data
+  cluster_name                       = "liran-demo"
```

Comment:
If you keep this example.  I don't know for what. Please update the readme with information about this example.

Also use only var for thinks like that

Reply:
16ee16acf8c3c4f0b2b43aa34ee22ce195b707ef

it's not urgent to update the README currently I think...

Reply:
[07534aa](https://github.com/env0/k8s-modules/pull/32/commits/07534aa9ccb02062551747c7a633f794b8bb0266)